templates/cloudflared/_helpers.tpl

```diff
@@ -0,0 +1,24 @@
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "cloudflare-tunnel.name" -}}
+{{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" }}
+{{- end }}
```

Comment:
This looks like a nice-to-have, but I don't see if being referenced elsewhere. Maybe we should remove it for now.

Reply:
Done!